src/views/App.js

```diff
@@ -3,17 +3,49 @@ import './App.css';
 import ServiceContent from './ServiceContent';
 import TextField from '@material-ui/core/TextField';
 import { makeStyles } from '@material-ui/styles';
+import { Button } from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
 
 function App() {
+  let storedid = localStorage.getItem('steamId') ? localStorage.getItem('steamId') : '';
```

Comment:
Wouldn't this return false regardless of the conditional?

Reply:
It would return null  - which could get mapped to storage because of the other logic I have - I've refactored it to remove this conditional and not save the string if it's empty (hence the REGEX question, for later on down the line).